src/libraries/System.Security.Claims/ref/System.Security.Claims.cs

```diff
@@ -90,7 +90,7 @@ protected ClaimsPrincipal(System.Runtime.Serialization.SerializationInfo info, S
         public ClaimsPrincipal(System.Security.Principal.IIdentity identity) { }
         public ClaimsPrincipal(System.Security.Principal.IPrincipal principal) { }
         public virtual System.Collections.Generic.IEnumerable<System.Security.Claims.Claim> Claims { get { throw null; } }
-        public static System.Func<System.Security.Claims.ClaimsPrincipal> ClaimsPrincipalSelector { get { throw null; } set { } }
+        public static System.Func<System.Security.Claims.ClaimsPrincipal?>? ClaimsPrincipalSelector { get { throw null; } set { } }
```

Comment:
Did this previously return null funcs, and could those funcs previously return nulls, and it just wasn't annotated correctly?